docs/manual/common/guide/build/UpgradingAkkaPlay.md

```diff
@@ -0,0 +1,211 @@
+# Using newer versions of Akka and Play Framework
+
+Lagom relies on Akka and Play Framework for some of its implementation.
+
+The versions of Akka and Play that ship with Lagom should usually be sufficient to run a Lagom application. However, in some cases it can be useful to upgrade them while staying on the same version of Lagom. This is usually possible when Akka and Play maintains binary compatibility between releases.
+
+Bear in mind that neither Akka nor Akka HTTP allow [mixed versions](https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed). As a consequence, you must make sure you override the version for all the artifacts on your dependency tree in sync.
+
+## sbt
+
+When you are using sbt, you can force new versions of Akka and Play by adding the following to your `build.sbt`:
+
+```scala
+val akkaVersion = "2.5.31"
+val akkaHttpVersion = "10.1.12"
+val playVersion = "2.7.6"
```

Comment:
```suggestion
val akkaVersion = "2.6.11"
val akkaHttpVersion = "10.1.13"
val playVersion = "2.8.7"
```

I was planning to send a PR to update to more recent versions. 

I agree that this information will get obsolete once Lagom we release Lagom with new versions and one day it will even get behind the current version. Still I find it strange that we show how to downgrade it.